src/scss/3_components/form/search-form.scss

```diff
@@ -22,6 +22,10 @@
   max-width: 500px;
   margin-bottom: 1.5em;
 
+  &.search_results {
+    margin-bottom: 0; // On search results page, no bottom margin under search box.
```

Comment:
Does this apply across all media queries? If so, I reckon that's fine but would be good to get input from @neilblair or @David-Corbett to confirm.

Reply:
No, doesn't apply to all, just applies to contact search